quality-gate.manifest.json

```diff
@@ -0,0 +1,108 @@
+{
+  "$schema": "https://raw.githubusercontent.com/govuk-one-login/quality-gates/refs/tags/v0.1.0/schemas/schema.json",
+  "services": [
+    {
+      "quality-gates": [
+        {
+          "check-types": ["code style and linting", "secret scanning"],
+          "config": {
+            "file": ".github/workflows/code-quality.yml",
+            "name": "pre-commit",
+            "path": "jobs.run-pre-commit"
+          },
+          "phase": "pre-merge",
+          "provider": "GitHub"
+        },
+        {
+          "check-types": ["code style and linting"],
+          "config": {
+            "file": ".github/workflows/code-quality.yml",
+            "name": "Linting",
```

Comment:
Do you think it's worth specifying that this is Java linting only? I think it would be good as it then carries the implication that TS / CFN linting is done in the pre-commit job